src/Kralizek.Extensions.Configuration.AWSSecretsManager/Internal/SecretsManagerConfigurationSource.cs

```diff
@@ -8,21 +7,22 @@ namespace Kralizek.Extensions.Configuration.Internal
 {
     public class SecretsManagerConfigurationSource : IConfigurationSource
     {
-        public SecretsManagerConfigurationSource(AWSCredentials? credentials = null, SecretsManagerConfigurationProviderOptions? options = null)
+        public SecretsManagerConfigurationSource(AWSCredentials? credentials = null, SecretsManagerConfigurationProviderOptions? options = null, RegionEndpoint? region = null)
         {
             Credentials = credentials;
             Options = options ?? new SecretsManagerConfigurationProviderOptions();
+            Region = region;
         }
 
         public SecretsManagerConfigurationProviderOptions Options { get; }
 
-        public AWSCredentials? Credentials {get; }
+        public AWSCredentials? Credentials { get; private set; }
```

Comment:
Isn't `private set;` already implied here?

Reply:
In C# 6 If you want to set the value you have to have a private setter, otherwise the compiler will flag the auto generated field as readonly. Alternatively you need to change from the automatic backing field to a defined field.